### DISCUSSION TITLE
how to get a imputation gene score matrix

### DISCUSSION THREAD

**Discussion by tianchen2019** [GreenleafLab/ArchR#646] [Questions / Documentation]
i used getMatrixFromProject(b,useMatrix = "GeneScoreMatrix") to get gene score matrix,but i want to know how to get a imputation gene score matrix.

**Comment by rcorces** [comment]
you can impute matrices in ArchR using the `imputeMatrix()` function. See issue https://github.com/GreenleafLab/ArchR/issues/513
